### PR TITLE
Auton two ball

### PR DESCRIPTION
**Pull request by tonyn10**
Changed the path to start at tape line and make it drive more centered to fender, still needs testing. Added shuffleboard for auton data in Robot class.

### CODE REVIEW

File: src/main/java/frc/robot/commands/AutonModes.java [hunk 1]
```diff
@@ -19,7 +20,7 @@
 import java.io.IOException;
 import java.nio.file.Path;
 
-public class AutonModes {
+public class AutonModes extends SubsystemBase {
```
Comment (azvanderpas) [inline src/main/java/frc/robot/commands/AutonModes.java]: Why do we want it to be a subsystem?

**Review by azvanderpas — DISMISSED** [review]

File: src/main/java/frc/robot/commands/AutonModes.java (hunk 1 above)
Comment (tonyn10) [reply src/main/java/frc/robot/commands/AutonModes.java]: I'll fix that, it was a discontinuous idea.

File: src/main/java/frc/robot/commands/AutonModes.java [hunk 2]
```diff
@@ -192,13 +192,12 @@ private void initializeCommandGroups() {
 
       twoBallCommand =
           new SequentialCommandGroup(
-              new DeployIntake(intakeSubsystem, cdsSubsystem),
+              // new DeployIntake(intakeSubsystem, cdsSubsystem),
```
Comment (sarango2158) [inline src/main/java/frc/robot/commands/AutonModes.java:195]: I think we still want to deploy the intake in autonomous, was this just for testing?

**Review by sarango2158 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/AutonModes.java (hunk 2 above)
Comment (tonyn10) [reply src/main/java/frc/robot/commands/AutonModes.java:195]: The parallel command that has combined intake and cds will deploy it.

**Review by sarango2158 — APPROVED** [review]

**Pull request closed without merging**